packages/usa-header/src/index.js

```diff
@@ -22,7 +23,7 @@ const CLOSE_BUTTON = `.${PREFIX}-nav__close`;
 const OVERLAY = `.${PREFIX}-overlay`;
 const CLOSERS = `${CLOSE_BUTTON}, .${PREFIX}-overlay`;
 const TOGGLES = [NAV, OVERLAY].join(", ");
-const NON_NAV_ELEMENTS = `body > *:not(${HEADER}):not([aria-hidden])`;
+const NON_NAV_ELEMENTS = `body *:not(${HEADER}, ${NAV_CONTAINER}, ${NAV}, ${NAV} *):not([aria-hidden])`;
```

Comment:
Make most of the header visible to screen readers while making `.usa-navbar` (and all other page content) invisible.

Hiding `usa-navbar` prevents the user from accessing the page title and menu button while the menu is open.